sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableConfig.java

```diff
@@ -156,6 +162,12 @@ public BigtableConfig withEmulator(String emulatorHost) {
     return toBuilder().setEmulatorHost(emulatorHost).build();
   }
 
+  @VisibleForTesting
+  BigtableConfig withBigtableClientOverride(BigtableClientOverride clientOverride) {
+    checkArgument(clientOverride != null, "clientOverride can not be null");
```

Comment:
If someone has set the override and you want to clear it, how do you do that?

Reply:
good point, I guess this check isn't really necessary. I'll remove it

Reply:
on second thought, all of the other withParameter functions here have similar null checks that wouldn't allow overrides.

I'm leaning towards keeping this to be consistent. I don't think there's a good reason to ever set this and then clear it.

Reply:
OK that seems fine. It is backwards-compatible to start to allow nulls as a way to re-clear it. One reason to allow nulls is to allow someone to make a copy without forcing them to `if (!= null) { set it }` but just be able to do straight-line `set to same value, whether or not it is null`